sklearn/utils/tests/test_metaestimators.py

```diff
@@ -24,3 +33,33 @@ def test_delegated_docstring():
                 in str(MockMetaEstimator.func.__doc__))
     assert_true("This is a mock delegated function"
                 in str(MockMetaEstimator().func.__doc__))
+
+
+def test_stochastic_gradient_loss_param():
```

Comment:
This test doesn't really belong to test_meta_estimators.py maybe somewhere like sklearn/tests/test_grid_search.py would be better.